runtime/src/accounts_db.rs

```diff
@@ -841,6 +842,25 @@ impl AccountsDB {
         infos
     }
 
+    pub fn verify_zero_lamports(&self, _slot: Slot, ancestors: &HashMap<Slot, usize>) -> bool {
```

Comment:
This moved from [here](https://github.com/solana-labs/solana/pull/7013/files#diff-a7549f152920d85fb44e6a784b8e5e1fL1394).